### DISCUSSION TITLE
OpenTelemetry support

### DISCUSSION THREAD

**Discussion by bhaskarbanerjee** [aws-powertools/powertools-lambda-python#5458] [Q&A]
Is support for manual or auto instrumentation of this package available through OpenTelemetry?

**Comment by leandrodamascena** [comment]
Hi @bhaskarbanerjee, thanks for your interest in this topic and for opening this discussion. Also, sorry for the late reply.

We are planning to create an RFC to support OpenTelemetry in 2025, but we need to start the discussion, write the RFC, and implement the initial DX, so we have a long way to go. When we open the RFC, we will pin the issue for feedback.

**Comment by leandrodamascena** [comment]
I'm closing this discussion as resolved. Please reopen if you need.
